pkg/render/logstorage/components/elasticsearch_metrics.go

```diff
@@ -0,0 +1,151 @@
+package component
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/tigera/operator/pkg/render/common/secret"
+
+	"github.com/tigera/operator/pkg/ptr"
+	relasticsearch "github.com/tigera/operator/pkg/render/common/elasticsearch"
+	appsv1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	operatorv1 "github.com/tigera/operator/api/v1"
+	"github.com/tigera/operator/pkg/components"
+	"github.com/tigera/operator/pkg/render"
+	rmeta "github.com/tigera/operator/pkg/render/common/meta"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"k8s.io/apimachinery/pkg/util/intstr"
```

Comment:
I've been trying to organize our imports into the 3 categories:
1. The golang imports (fmt, strings)
2. The non-operator imports (basically everything that isn't github.com/tigera/operator)
3. The github.com/tiger/operator imports